reactor-netty-http/src/main/java/reactor/netty/http/client/HttpClientSecure.java

```diff
@@ -29,7 +29,7 @@
  * @author Stephane Maldini
  * @author Violeta Georgieva
  */
-final class HttpClientSecure {
+public final class HttpClientSecure {
```

Comment:
if we want to avoid turning that class public, we can always have a public final field in another class (eg. `HttpClient`) that is an alias for `ENABLE_HOSTNAME_VERIFICATION` (like `static final Consumer ALIAS = HttpClientSecure.ENABLE_HOSTNAME_VERIFICATION`). Or even move it altogether. wdyt?

Reply:
extracted this configurer to a new class